package.json

```
@@ -26,15 +26,18 @@
     "node": ">=10.0.0"
   },
   "dependencies": {
-    "postcss": "^7.0.27",
-    "postcss-values-parser": "^3.2.0"
+    "postcss-values-parser": "^4.0.0"
```

```suggestion
    "postcss-values-parser": "^5.0.0"
```

Tested locally and tests also pass with the latest version